src/cargo/sources/registry/index/mod.rs

```diff
@@ -869,6 +886,10 @@ impl<'a> RegistryDependency<'a> {
             dep.set_artifact(artifact);
         }
 
+        let features = features
```

Comment:
collect seems not needed? `set_features` accepts iterator.

Reply:
Good catch! Thank you!